conf/test_settings.py

```diff
@@ -12,7 +12,7 @@
 
 ENGINE_RPC_URL = "http://INTENTIONALLY_DISCONNECTED:9999"
 INTERNAL_URL = "http://INTENTIONALLY_DISCONNECTED:9999"
-PROFILES_URL = "http://INTENTIONALLY_DISCONNECTED:9999"
+PROFILES_URL = "http://intentionally_disconnected:9999"
```

Comment:
Is this a necessary change?

Reply:
This was for the assertion checks, as when checking the assertions mocked by httpretty it has it in lowercase.